### PR TITLE
Automate build of k3s images

### PR DESCRIPTION
**Pull request by eveld**
Every day, the latest built version is compared to the latest released k3s version. If there is a new k3s version, a new packer build is triggered for that version of k3s.

### CODE REVIEW

**Comment by eveld** [comment]
After this is accepted, we need to update the GCP project ID and SA to the correct ones.

File: .github/workflows/build.yml [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+        id: version
+        run: |-
+          # Get latest K3s version
+          K3S_VERSION=${{inputs.version}}
+          if [ -z "$K3S_VERSION" ]; then
+            K3S_VERSION=$(curl -s https://api.github.com/repos/k3s-io/k3s/releases | \
+              jq -r '.[].tag_name | select(contains("-rc") | not)' | \
+              sort -V -r | \
+              head -n 1)
+            CURRENT_VERSION=$(curl --silent --fail -L \
```
Comment (amochtar) [inline .github/workflows/build.yml:28]: If we create a tag for every build, shouldn't we just check whether the tag for `$K3S_VERSION` already exists?

**Review by amochtar — COMMENTED** [review]
Looks good, just a small comment on the check if a build already exists.

**Comment by amochtar** [comment]
Oh, we might want to use this: https://github.com/marketplace/actions/get-latest-release. Could simplify the workflow quite a bit.

**Comment by amochtar** [comment]
Closed by #8

**Pull request closed without merging**